media-video/handbrake/patches/handbrake-1.0.7.patchset

```diff
@@ -4227,3 +4227,230 @@ index 44da58a..0000000
 -- 
 2.10.2
 
+From: Ken Mays <kmays2000@gmail.com>
+Date: Fri Aug 10 13:01:42 GMT 2018 
+Subject: fix for FFmpeg 4.x dependency
+
+
+--- HandBrake-1.0.7/libhb/encavcodec.c.old	2018-08-10 11:50:45.138674176 +0000
++++ HandBrake-1.0.7/libhb/encavcodec.c	2018-08-10 11:52:56.821559296 +0000
+@@ -235,7 +235,7 @@
+         /* Constant quantizer */
+         // These settings produce better image quality than
+         // what was previously used
+-        context->flags |= CODEC_FLAG_QSCALE;
++        context->flags |= AV_CODEC_FLAG_QSCALE;
+         context->global_quality = FF_QP2LAMBDA * job->vquality + 0.5;
+         //Set constant quality for libvpx
+         if ( w->codec_param == AV_CODEC_ID_VP8 ||
+@@ -278,11 +278,11 @@
+ 
+     if( job->mux & HB_MUX_MASK_MP4 )
+     {
+-        context->flags |= CODEC_FLAG_GLOBAL_HEADER;
++        context->flags |= AV_CODEC_FLAG_GLOBAL_HEADER;
+     }
+     if( job->grayscale )
+     {
+-        context->flags |= CODEC_FLAG_GRAY;
++        context->flags |= AV_CODEC_FLAG_GRAY;
+     }
+ 
+     if( job->pass_id == HB_PASS_ENCODE_1ST ||
+@@ -302,7 +302,7 @@
+                 ret = 1;
+                 goto done;
+             }
+-            context->flags |= CODEC_FLAG_PASS1;
++            context->flags |= AV_CODEC_FLAG_PASS1;
+         }
+         else
+         {
+@@ -341,7 +341,7 @@
+             fclose( pv->file );
+             pv->file = NULL;
+ 
+-            context->flags    |= CODEC_FLAG_PASS2;
++            context->flags    |= AV_CODEC_FLAG_PASS2;
+             context->stats_in  = log;
+         }
+     }
+--- HandBrake-1.0.7/libhb/muxavformat.c.old	2018-08-10 11:54:00.651165696 +0000
```

Comment:
Where does the "old" postfix comes from?

Reply:
probably they patched it by moving files around, not via the built in HaikuPorter git repo. It won't cause any harm, just not the preferred way